src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/ElasticSearchDAOSupport.scala

```diff
@@ -49,7 +49,7 @@ trait ElasticSearchDAOSupport extends LazyLogging {
       case (label: String, detail: AttributeDetail) => detailFromAttribute(label, detail)
     }
     // add the magic "_suggest" property that we'll use for autocomplete
-    val props = attributeDetailMap + ((fieldSuggest, ESType.suggestField("string")))
+    val props = attributeDetailMap + ((fieldSuggest, ESType.suggestField("string"))) + ((fieldDiscoverableByGroups, ESInternalType("string")))
```

Comment:
i'm not sure what the recommended way to do this is. i need the definition of fieldDiscoverableByGroups for this code to compile. I could have done a diff from my 1507. do you have a suggestion?